### PR TITLE
Labs Colouring

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
Adds colours for `Special.Labs` articles

### Before
Screenshot 2021-03-16 at 17 50 20

### After
Screenshot 2021-03-17 at 10 25 29

## Why?
More greys and blacks

### CODE REVIEW

**Comment by oliverlloyd** [comment]
@ioanna0 @HarryFischer @mxdvl thanks for helping on this!

**Comment by oliverlloyd** [comment]
[Labs colouring (via palette)](https://trello.com/c/OAwbb8Dd/2660-labs-colouring-via-palette)

File: src/web/lib/decidePalette.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ import {
 import { pillarPalette } from '@root/src/lib/pillars';
 
 const WHITE = neutral[100];
-const BLACK = text.primary;
+const BLACK = neutral[0];
```
Comment (HarryFischer) [inline src/web/lib/decidePalette.ts:22]:
```suggestion
const BLACK = neutral[7];
```
My bad here - neutral.0 is #000. We wanted neutral.7 #121212

File: src/web/lib/decidePalette.ts [hunk 2]
```diff
@@ -349,6 +358,7 @@ const backgroundArticle = (format: Format): string => {
 	if (format.design === Design.Comment) return opinion[800];
 	if (format.design === Design.Editorial) return opinion[800];
 	if (format.theme === Special.SpecialReport) return specialReport[800]; // Note, check theme rather than design here
+	if (format.theme === Special.Labs) return neutral[86];
```
Comment (HarryFischer) [inline src/web/lib/decidePalette.ts:361]:
```suggestion
	if (format.theme === Special.Labs) return neutral[97];
```
Doms new designs use this bg colour instead for higher contrast

**Review by HarryFischer — CHANGES_REQUESTED** [review]
Couple colour tweaks to match new (accessible) designs

**Review by HarryFischer — APPROVED** [review]
👍

**Review by liywjl — APPROVED** [review]
LGTM 👍

**Pull request merged**